src/eos-autoupdater.c

```diff
@@ -65,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *LOCAL_CONFIG_FILE_PATH = PREFIX "/local/share/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
Using `PREFIX` means it works when build on Debian (`--prefix=/usr`) but when built under JHBuild (`--prefix=/`) we end up with `/local/share/eos-updater/…`, which isn’t great. I can’t think of a decent way of handling this without adding another `configure.ac` argument which essentially replicates `--prefix` and pretends you never changed it from `/usr/local` though.